lib/puppet/pops/lookup/explainer.rb

```diff
@@ -345,7 +345,7 @@ def dump_on(io, indent, first_indent)
       indent = increase_indent(indent)
       if @provider.respond_to?(:config_path)
         path = @provider.config_path
-        io << indent << 'Using configuration "' << path.to_s << "\"\n" unless path.nil?
+        io << indent << _('Using configuration "') << path.to_s << "\"\n" unless path.nil?
       end
       branches.each {|b| b.dump_on(io, indent, indent)}
       dump_outcome(io, indent)
```

Comment:
I'm inclined to say we shouldn't externalize this file... I think the result might be a mix of things that's unintelligible due to the way we're concatenating various strings.